_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -194,6 +254,7 @@ pre-configure {
         ui_debug "Adding -DCMAKE_INSTALL_RPATH=[join ${cmake.install_rpath} \;] to configure.args"
         configure.args-append -DCMAKE_INSTALL_RPATH="[join ${cmake.install_rpath} \;]"
     }
+    configure.pre_args-prepend "-G \"[join ${cmake.generator}]\""
```

Comment:
Probably easier to turn `cmake.generator` from a list-value into a string value so that you can drop the extra `[join ...]` here. That could be done if you expected users to quote `Unix Makefile` when passing it to `cmake.generator`.